### PR TITLE
Feat/dimon workflow

### PR DESCRIPTION
**Pull request by e-dimon17**
Implemented subnet basic workflow for random wiki-scraper articles.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (Netel-pleb) [inline docs/quickstart.md:1]: This doesn't make sense.
100% should be updated right now.

File: docs/running_on_testnet.md [hunk 2]
```diff
@@ -22,10 +22,23 @@ CREATE ROLE your_user_name WITH LOGIN SUPERUSER PASSWORD 'your_password';
 brew services restart postgresql@14
```
Comment (Netel-pleb) [inline docs/running_on_testnet.md:22]: This docs too, should be updated.
Reference other subnets.

File: docs/quickstart.md (hunk 1 above)
Comment (e-dimon17) [reply docs/quickstart.md:1]: Yeah, I will update it. thank you

File: docs/running_on_testnet.md (hunk 2 above)
Comment (e-dimon17) [reply docs/running_on_testnet.md:22]: I will made it to read the configurations like password and user_role etc from .env or config.yaml file.

**Pull request merged**